prisma-fmt/src/get_config.rs

```diff
@@ -242,9 +240,7 @@ mod tests {
                 "DBURL": "postgresql://example.com/mydb",
             }
         });
-        let expected = expect![[
-            r#"{"message":"\u001b[1;91merror\u001b[0m: \u001b[1mEnvironment variable not found: DOES_NOT_EXIST.\u001b[0m\n  \u001b[1;94m-->\u001b[0m  \u001b[4mschema.prisma:5\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\u001b[1;94m 4 | \u001b[0m                url = env(\"DBURL\")\n\u001b[1;94m 5 | \u001b[0m                directUrl = \u001b[1;91menv(\"DOES_NOT_EXIST\")\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\nValidation Error Count: 1","error_code":"P1012"}"#
-        ]];
+        let expected = expect![[r#"{"message":"\u001b[1;91merror\u001b[0m: \u001b[1mEnvironment variable not found: DOES_NOT_EXIST.\u001b[0m\n  \u001b[1;94m-->\u001b[0m  \u001b[4mschema.prisma:5\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\u001b[1;94m 4 | \u001b[0m                url = env(\"DBURL\")\n\u001b[1;94m 5 | \u001b[0m                \u001b[1;91mdirectUrl = env(\"DOES_NOT_EXIST\")\u001b[0m\n\u001b[1;94m   | \u001b[0m\n\nValidation Error Count: 1","error_code":"P1012"}"#]];
```

Comment:
The diff in this example is that before only the `env(...)` part was one format, and now the whole line it??

Reply:
I'm honestly not sure how to read the diff here 😅

Reply:
<img width="1526" alt="image" src="https://user-images.githubusercontent.com/183673/222780236-959bd181-52df-4433-a7a2-a1ea64a6d690.png">

I think this is the additional line with a line number and the `|` that was removed - I commented on that below as well in a snapshot where it is more clearly visible.

Reply:
(I think in `prisam/prisma` we use a stripAnsi function to remove the color codes from the messages and snapshot that as well - as it is more readable without breaking your brain)